spanner/client_test.go

```diff
@@ -2971,66 +3067,68 @@ func TestBatchReadOnlyTransactionFromID_ReadOptions(t *testing.T) {
 }
 
 type QueryOptionsTestCase struct {
-	name   string
-	client QueryOptions
-	env    QueryOptions
-	query  QueryOptions
-	want   QueryOptions
+	name      string
+	client    QueryOptions
+	clientDRO *sppb.DirectedReadOptions
+	env       QueryOptions
```

Comment:
@rahul2393 
Should we create a new struct for Directed Read Options instead of modifying the existing `QueryOptionsTestCase`? If we write a new struct then there would be new methods added which would look similar to  [testQueryOptions](https://github.com/googleapis/google-cloud-go/blob/1b0d41cb6b63daee3bc707515d8c301b5dc40320/spanner/client_test.go#L452) and [checkReqsForQueryOptions](https://github.com/googleapis/google-cloud-go/blob/1b0d41cb6b63daee3bc707515d8c301b5dc40320/spanner/client_test.go#L463).
Which one would be the right way to move with?

Reply:
Using the same struct to avoid code duplication